src/dev/env.md

```diff
@@ -53,3 +53,10 @@ only intended for use in testing.
 
 This variable is only available when building with the `yk_testing` Cargo
 feature is enabled.
+
+### `YKD_TRACE_DEBUGINFO`
+
+When `YKD_TRACE_DEBUGINFO=1`, the JIT will add debugging information to JITted
+traces, allowing debuggers conforming to the [gdb JIT
+interface](https://sourceware.org/gdb/current/onlinedocs/gdb/JIT-Interface.html)
+to show higher-level representations of the code in the source view.
```

Comment:
We should probably add a warning along the lines of "Warning: this debugging information requires the use of temporary files which are slow to create and may not always be cleaned up." [I might not have got the warning right, but just as an example.]

Reply:
Whilst true, I wonder if we should add a "blanket warning" at the top of the page, as just about every `YKD_*` variable impacts performance in some way. Thoughts?

Reply:
This one isn't just about performance: it's also about cleanup (or not).

Reply:
Try ef97011